dialog/form_test.go

```diff
@@ -0,0 +1,144 @@
+package dialog
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/test"
+	"fyne.io/fyne/widget"
+	testify "github.com/stretchr/testify/assert"
```

Comment:
We generally just import it as `assert` (without changing import name) and then use the regular assert functions instead of the `.New(t)` and then calling methods on that. That applies in more parts of the code, just mentioning it here.

Reply:
I can change that. Is there a reason `require` isn't used?